app/src/layouts/common/appSidebar/appSidebar.jsx

```diff
@@ -20,6 +20,12 @@ import classNames from 'classnames/bind';
 import { referenceDictionary } from 'common/utils';
 import { useIntl, defineMessages } from 'react-intl';
 import { Sidebar } from 'componentLibrary/sidebar';
+import { ServiceWithPopover } from 'layouts/common/appSidebar/helpAndService';
+import LogoHelp from 'common/img/help-inline.svg';
+import { useSelector } from 'react-redux';
+import { userIdSelector } from 'controllers/user';
+import { getFAQOpenStatus } from 'controllers/log/storageUtils';
```

Comment:
Please do not mix the storage utils for different purposes under the not related directories.
It is best to create a separate file for `faq` related utilities. It can be stored near the sidebar, as they will not be used elsewhere.